CHANGELOG.asciidoc

```diff
@@ -14,8 +14,7 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
 - Libbeat HTTP's Server can listen to a unix socket using the `unix:///tmp/hello.sock` syntax. {pull}13655[13655]
 - Libbeat HTTP's Server can listen to a Windows named pipe using the `npipe:///hello` syntax. {pull}13655[13655]
 - Adding new `Enterprise` license type to the licenser. {issue}14246[14246]
-
-*Auditbeat*
+- By default, all Beats-created files and folders will have a umask of 0027 (on POSIX systems). {pull}14119[14119]
```

Comment:
Is this true for all Beats? The PR made me think that individual Beats can override the default.